tests/support/mod.rs

```diff
@@ -1,3 +1,8 @@
+// Using a shared mod like this is probably not the best idea, since we have to
+// disable the `dead_code` lint, as we don't need all of the helpers from here
+// all over the place.
+#![allow(dead_code)]
```

Comment:
I'm a bit confused. If we have dead code why do we not just remove it?

Reply:
The problem is this mod is used in tests, and each test is an individual compile unit. So, `tests/buffering.rs` complains about one set of fns as dead code, and `tests/topology.rs` doesn’t like the other. This is kind of a bad pattern to organize test helpers like that, but here we are.

Reply:
The problem is this mod is used in tests, and each test is an individual compile unit. So, `tests/buffering.rs` complains about one set of fns as dead code, and `tests/topology.rs` doesn’t like the other. This is kind of a bad pattern to organize test helpers like that, but here we are.

Reply:
@Hoverbear we'll need to refactor it at some point. I don't want to do it as part of this PR though since it's sort of a hotfix, and we better merge is asap.

Reply:
Agreed. :)